AudibleBoulders/AudibleBoulders
# Update post-commit script to pull info for schema 2.0

#### What's this PR do?
Post-commit script pulls additional info from local repo to match schema 2.0
Update name of USER_KEYS.js to SIGNATURE_HASH.js and fixed text in setup.html page
Update gitSpy.zip to match changes

#### What are the important parts of the code?
DEV-gitSpy folder
src/client/gitSpy.zip

#### How should this be tested by the reviewer?
Unzip file and make sure the zip file is up-to-date with DEV-gitSpy folder
Run server and look at setup page to make sure the signature_hash has " "'s around it

#### Is any other information necessary to understand this?
N/A

#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
N/A

#### Screenshots (if appropriate)
N/A
